apps/worker/src/app/workflow/usecases/send-message/send-message-push.usecase.ts

```diff
@@ -43,6 +44,8 @@ const LOG_CONTEXT = 'SendMessagePush';
 export class SendMessagePush extends SendMessageBase {
   channelType = ChannelTypeEnum.PUSH;
 
+  private pushFactory;
```

Comment:
@p-fernandez due to how nestjs works, we cannot have this one here as a private method, as nestjs @Injectable classes are singletons, and this will be reused across multiple executions. Which we don't want We need here to not store it as a method but rather pass it.